FusionIIIT/applications/gymkhana/urls.py

```diff
@@ -0,0 +1,2 @@
+# from django.conf.urls import url
+# from . import views
```

Comment:
@3Peers/gymkhanateam , please add few dummy urlpatterns so that there is no error while running.